modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -1484,16 +1484,14 @@ The JSON input payload looks like the following.
 [[skip_on_null]]
 ==== Skip Null On (for JSON)
 
-You can specify whether this generates an outbound message that contains fields
-with "null" values, or if these fields are ignored entirely. This can be set
-through an attribute in the output directive named `skipNullOn`, which can be
-set to three different values: `elements`, `attributes`, or `everywhere`.
+You can use the `skipNullOn` writer property to omit null values from arrays, objects, or both.
 
 When set to:
 
-* `elements`: A key:value pair with a null value is ignored.
-* `attributes`: An XML attribute with a null value is skipped.
-* `everywhere`: Apply this rule to both elements and attributes.
+* `arrays`: Ignore and omit `null` values from JSON output. Example: `output application/json skipNullOn="arrays"`
+* `objects`: Ignore an object that has a null value. The output contains an empty object (`{}`) in place of the object with the null value. Example: `output application/json skipNullOn="objects"`
```

Comment:
ditto
```suggestion
* `objects`
+
+ Ignore an object that has a null value. The output contains an empty object (`{}`) instead of the object with the null value, for example, `output application/json skipNullOn="objects"`.
```